docs/migrations.md

```diff
@@ -171,20 +169,24 @@ def backward_mapper(page_or_revision, data):
 
 def forwards(apps, schema_editor):
     page_types_and_fields = [
-        ('myapp', 'MyPage', 'streamfield_name', 'streamblock_type'),
+        ('myapp', 'MyPage', 'field_name', 'block_type'),
```

Comment:
The added explanation below helps, but since this is in the context of our special Wagtail migration methods, I would leave this as `streamfield_name` to make it very clear what that param is expecting.

Reply:
What I'm trying to make clear is that it's the name of the field on the Django model — coming back to this to write a data migration after six months or so it took me a while to realize this.

Reply:
I suspected that's what you would say :) I'm fine with that.

Reply:
Heh. I'm totally open to making it clearer in any way possible — like I said, what I originally wrote was totally not obvious to me now :)